### ISSUE TITLE
Data import help

### ISSUE THREAD

**Issue by crass00** [bpepple/metron#10]
I've been adding a few entries and the thing that is going to make it difficult to get lots of data in is how slow I am.

I thought perhaps that it might help if when adding a new comic the creator data gets copied over as it is often the same or simular as the previous issue.  I see your after the the wider range of creators (ie editors etc) which can add up to a lot of people to manually add.

So for instance lets say I'm editing issue 5, then I want to add issue 6, I click the New Issue button* and some of the data is filled in already.  Perhaps the issue and the series data but it's the Creator data that is really slow for me to import.

The concern I do see here is editor lazyness leaving data that has changed in from issue to issue, but if it helps to streamline the process then that could get more people contributing.  Something that may mitigate editor lazyness here :) is although all the creators are prepolulated, have a confirm button next to each entry so the editor needs to make an active descision.

Alternativly/As well as, are there any other ways to ingest data apart from the GUI,  could CSV/json files be uploaded to create issues?

*Perhaps New Issue button could stay as is and a "Next Issue" button be added that does the pre-populating

**Comment by bpepple** [comment]
Agreed, adding an issues with a lot of creators or characters is quite time consuming. I've been thinking about this problem for awhile and haven't come up with a solution that I'm totally satisfied with. As far as I'm aware, Comic Vine or the Grand Comics Database haven't come up with a good solution for this problem either.

As far as the most palatable solution to implement it's pretty much the one you've outlined, which does have some drawbacks (mainly due to the user not verifying the information before submitting it), but it might merit testing out in a branch to see how well it works in practice.

This general problem is why thus far I haven't added a series type for Trade Paperbacks, since I'd like it to have the user select the issues included in the TPB and have it add the creators/characters automatically.

**Comment by bpepple** [comment]
I've done tangential work on this, and written a tool to fetch data and format it so it can be used with django's shell function to create new issues.

Still haven't come up with a decent solution for a more end-user solution, so I'll keep this open until I do.

**Comment by rad96** [comment]
uploading specially crafted json and parsing it to form for verification would be "good enough"

**Comment by bpepple** [comment]
Fixed in #411
